github/repos.go

```diff
@@ -146,6 +151,10 @@ func (s *RepositoriesService) List(user string, opt *RepositoryListOptions) ([]R
 		return nil, nil, err
 	}
 
+	if opt != nil && opt.IncludeOrg {
+		req.Header.Set("Accept", "application/vnd.github.moondragon+json")
```

Comment:
would you mind pulling this value out into a const alongside [similar values](https://github.com/google/go-github/blob/4da7e49b0b4273d0becd418b24a2f49f5c56bd66/github/github.go#L38-L44).  Also, go ahead and update the test to pass true, and test for the presence of the header using `testHeader`.